### ISSUE TITLE
Icon Request: fa-american-sign-language-interpreting and fa-sign-language

### ISSUE THREAD

**Issue by jackfd53** [FortAwesome/Font-Awesome#2026]
This would be very useful to place in event listings etc that are ASL Interpreted...I have to fool with an image when doing this on several church websites I'm working on.  Thank you for your consideration.

![off-black-terp-hands-md](https://f.cloud.github.com/assets/3398180/1206899/72dbb410-25a1-11e3-8a12-330dc48ebf56.png)

**Comment by hlwaymack** [comment]
I would also find having this icon added extremely useful.  I work on websites for community events and many of the events are ASL interpreted.  Being able to use the icon as part of the standard fontawesome arsenal would great!

**Comment by imagineertobe** [comment]
+1 except that I would suggest a more international version of the sign, as attached here, since the icon above is unique to American Sign Language and doesn't occur in most European sign languages, for instance. Iterations of the icon below is more commonly used throughout the world to reflect sign language use.
![circle-signlanguage-f37748](https://cloud.githubusercontent.com/assets/5739754/7636814/902ed1ac-fa37-11e4-9f6d-aaf74ca4bece.png)

**Comment by outsidediagonal** [comment]
+1

**Comment by truls1502** [comment]
+1 on @imagineertobe 

If they want ASL-icons, they could have a separate icon (such as fa-asl). Then @imagineertobe suggestion such as fa-signlanguage or fa-sl

**Comment by MVcodeartbr** [comment]
+1

**Comment by joe-bligh** [comment]
+1 , including British / International version please!

**Comment by AndreLGava** [comment]
+1

**Comment by netAction** [comment]
+1

**Comment by caduvieira** [comment]
Other version:
![image](https://cloud.githubusercontent.com/assets/3831408/14290142/3b494a4c-fb34-11e5-9100-e2494a0c555b.png)

https://www.ufmg.br/marca/libras/

**Comment by netAction** [comment]
The German Sign Language has an own consitent icon. It can be used with or without the text DGS.

![csm_dgs-symbol_dunkelgrau_invers_80e98ef478](https://cloud.githubusercontent.com/assets/1829847/14312134/2f021cca-fbeb-11e5-8eb9-77c00df3a8f9.png)

**Comment by davegandy** [comment]
Added to 4.6.0-wip branch. Closing.

**Comment by caduvieira** [comment]
Could we use the new 'add your reaction' in github to vote for which icon in this issue we would like to be implemented?

**Comment by netAction** [comment]
Usually it should be the typographer's choice. In this case there are standards and Dave decided to support the US version. So I see no request where a poll could make sense.

Now we need a new issue for the international (and simpler/clearer) version of this symbol.

**Comment by truls1502** [comment]
Hi @netAction 

For ASL (American Sign Language) which is US version, I agree with the sign for interpreting in https://github.com/FortAwesome/Font-Awesome/issues/2026#issue-20022685.

However, the international version for either "Sign Language Interpreting" or "Sign Language", I strongly recommend to use the icon from this https://github.com/FortAwesome/Font-Awesome/issues/2026#issuecomment-102100667

**Comment by netAction** [comment]
Yes that's right. So this issue is not solved. We shoud reopen it.

**Comment by davegandy** [comment]
@netAction Apologies! I thought we already re-opened this issue.

**Comment by davegandy** [comment]
Is there a standard icon internationally for sign language interpreting / sign language? Does this one get it? https://github.com/FortAwesome/Font-Awesome/issues/2026#issuecomment-102100667

**Comment by statik** [comment]
Yes, that one is excellent and is the universal/international symbol for sign language, it is being adopted in sign languages all over the world as the gesture that represents any signed language.

**Comment by truls1502** [comment]
@statik, I am unfortunately not agree with you! I have attend a lot different international conference, seminar etc. We have different sign for "Interpreting", the sign looks like this http://www.handspeak.com/world/isl/index.php?id=208. However you have a bit right that the sign is powerful around the world - but is not correct sign.

@davegandy I asking you nice to change fa-sign-language-interpreting to fa-america-sign-language-interpreting or fa-america-signlanguage-interpreting. (alias fa-asl-interpreting). Hopefully we will get a own international sign for "interpreting" later.

**Comment by davegandy** [comment]
@truls1502 I'll change it to fa-american-sign-language-interpreting and alias to fa-asl-interpreting. I'm also going to see if I can fit in fa-sign-language (from here: https://github.com/FortAwesome/Font-Awesome/issues/2026#issuecomment-102100667) which I will also create from scratch to make sure visually fits with Font Awesome.

Thanks for the clarification and for the patience!

**Comment by statik** [comment]
@truls1502 I agree with you, I am thinking of http://www.handspeak.com/world/isl/index.php?id=243 which means sign language rather than interpreting. Super awesome to see this work happening, and I think @davegandy current plan is excellent.

**Comment by truls1502** [comment]
@statik I am really sorry, I thought you meant "interpreting" than "sign language". I am apologize. Yes, I am looking forward it to happening, finally it start now and hopefully the 4.6 version will release soon as possible.

@davegandy thank you! The icon "sign" on https://github.com/FortAwesome/Font-Awesome/issues/2026#issuecomment-206257800 is as well good. It is only preference how it looks. But if I am able to combine such as stacked Icons - it would be great.

**Comment by imagineertobe** [comment]
I would like to clarify a couple of things just so that we're all on the same page:

- The image referenced in https://github.com/FortAwesome/Font-Awesome/issues/2026#issue-20022685 is the American Sign Language sign for "interpret" and is the most common icon seen in places of public accommodation to indicate that events will be interpreted into and from ASL.
- The image referenced in my previous post (https://github.com/FortAwesome/Font-Awesome/issues/2026#issuecomment-206257800) is a common sign for "sign" across several European sign languages and was adopted as the sign for "sign" in International Sign/Gestuno. As an icon, it is commonly seen in places of public accommodation to indicate that events will be given in sign language (sometimes via an interpreter, sometimes directly by a sign language user).

There are legal reasons for the difference between the usages, largely that deaf people have fewer legal rights to requesting interpreter services in most European countries than American deaf people do in the US.

The point I'm emphasizing here is that the ASL icon is indeed about interpreting, whereas the European-inspired/"more international" icon is about the use of sign language in general, not necessarily via an interpreter. The Font Awesome short codes should reflect these facts.

**Comment by davegandy** [comment]
@truls1502 Yep, you can definitely stack these the same way as in the [examples](http://fontawesome.io/examples/#stacked). 

@imagineertobe I am doing both icons. There is already an `fa-american-sign-language-interpreting` in 4.6.0-wip branch. I am also doing the internationally recognized `fa-sign-language` (https://github.com/FortAwesome/Font-Awesome/issues/2026#issuecomment-102100667). 

Folks, if any of this doesn't look right, or should be done differently, please let me know soon. I'm hoping to get this release out in the next few days.

**Comment by davegandy** [comment]
Added to 4.6.0-wip branch. Closing.
<img width="767" alt="screenshot 2016-04-07 16 27 22" src="https://cloud.githubusercontent.com/assets/1076721/14367433/b5838aac-fcdd-11e5-980f-9efe99c3b38c.png">
